x11-libs/libxcursor/libxcursor-1.1.15.recipe

```diff
@@ -35,30 +38,28 @@ REQUIRES_devel="
 
 BUILD_REQUIRES="
 	haiku${secondaryArchSuffix}_devel
-	devel:cursorproto$secondaryArchSuffix
-	devel:fixesproto$secondaryArchSuffix
-	devel:kbproto$secondaryArchSuffix
-	devel:libpthread_stubs$secondaryArchSuffix
-	devel:libX11$secondaryArchSuffix
+	#devel:cursorproto
+	devel:fixesproto
+	devel:kbproto
+	devel:libpthread_stubs
+	devel:libx11$secondaryArchSuffix
+	devel:libxau$secondaryArchSuffix
+	devel:libxcb$secondaryArchSuffix
+	devel:libxdmcp$secondaryArchSuffix
```

Comment:
```diff
-	devel:libxau$secondaryArchSuffix
-	devel:libxcb$secondaryArchSuffix
-	devel:libxdmcp$secondaryArchSuffix
```

Reply:
Package 'xcb', required by 'x11', not found ...